src/components/TodoItem/todoItem.scss

```diff
@@ -0,0 +1,10 @@
+.todoitem {
```

Comment:
Isn't it necessary to add a dash between `todo` and `item`? Just to be consistent with the naming convention.